src/redis_mod.F90

```diff
@@ -0,0 +1,455 @@
+MODULE redis_mod
+  USE iso_c_binding
```

Comment:
Note: this file was renamed (suffix: `F90`) so its showing that all the lines were changed. I can revert this change if you want.

Reply:
we should go back to .f90 I think. This is causing a build error on travis, but probably not on your laptop bc mac is not case sensitive.